devtools/create_project/create_project.cpp

```diff
@@ -431,6 +433,12 @@ int main(int argc, char *argv[]) {
 		setup.libraries.push_back("sdl2");
 	}
 
+	if (setup.useCanonicalLibNames) {
+		for (auto& lib : setup.libraries) {
```

Comment:
auto variables are not allowed in ScummVM yet. Please, specify the correct type and use conventional `for` loop